src/wombats/lib/wombat_lib.js

```diff
@@ -0,0 +1,282 @@
+// Have module.exports be a function so that user won't have to worry about namespaces
+module.exports = function() {
+  this.turn_directions = ['right', 'left', 'about-face'];
```

Comment:
I'm not super familiar with using `exports` like this, but can you use `const` instead so that the variables are immutable?